src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -186,16 +190,6 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
 
       return items;
     },
-
-    [FilterType.NAMESPACE]: items => {
```

Comment:
This is not needed or why is this removed?

Reply:
Because `Items` don't necessarily have a `getNs()` command, only `KubeObjects` do. So this was moved to `KubeObjectListLayout` and passed down.